src/elements/fields/DropdownField.tsx

```diff
@@ -52,7 +53,7 @@ export default function DropdownField({
   }, [countryCode, setCurCountry]);
 
   let options;
-  if (dynamicOptions.length > 0) {
+  if (servar.metadata.salesforce_sync && !editMode) {
```

Comment:
maybe this should be a boolean returned from the hook as well, like `shouldSyncSalesforce`